Tests/TestCodeGenConfigurations/Other-CocoaPods/CocoaPodsProject.xcodeproj/project.pbxproj

```diff
@@ -704,6 +704,7 @@
 				MTL_ENABLE_DEBUG_INFO = INCLUDE_SOURCE;
 				MTL_FAST_MATH = YES;
 				ONLY_ACTIVE_ARCH = YES;
+				OTHER_SWIFT_FLAGS = "\"-enable-upcoming-feature ExistentialAny\"";
```

Comment:
Doesn't this one need the enclosing `"` removed too? There is one more at [L763](https://github.com/apollographql/apollo-ios-dev/pull/379/files#diff-55e3f5e7faa355914df4319b13344ca8cac090f7d15a748b24019a41dd395c82R763) that has them too.